cli/src/pcluster/templates/login_nodes_stack.py

```diff
@@ -235,6 +235,10 @@ def _add_login_nodes_pool_launch_template(self):
                         self._shared_storage_attributes[SharedStorageType.EFS]["IamAuthorizations"],
                         use_lower_case=True,
                     ),
+                    "efs_accesspoint_ids": to_comma_separated_string(
```

Comment:
This change should be captured in unit test here: https://github.com/aws/aws-parallelcluster/blob/develop/cli/tests/pcluster/templates/test_login_nodes_stack.py#L23